### PR TITLE
Logout

### PR DESCRIPTION
**Pull request by xiaoalem**
added log out button on dashboard.html and index.html; and added tripTitle to cache on tripDashboard

### CODE REVIEW

File: src/main/webapp/javascript/dashboard.js [hunk 1]
```diff
@@ -87,4 +91,16 @@ function getBudget(tripId) {
...
\ No newline at end of file
+}
+
+
+// Log out
+var logout = document.getElementById("logout-btn");
+var msgDiv = document.getElementById("status-msg");
+logout.addEventListener("click", function() {
+    sessionStorage.clear();
+    msgDiv.innerHTML = `<p color=red> Successully logged out</p>`
```
Comment (emmayuan25) [inline src/main/webapp/javascript/dashboard.js]: can we change `color=red` to `style="color: red;"`

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -11,15 +11,15 @@ if(username === "" || username === null){
...
-    location.href = "./pages/userSignup.html";
-}
-
-// Direct user to sign in page
-document.getElementById("userSignin").onclick = () => {
-    location.href = "./pages/userSignin.html";
+// Log out
+document.getElementById("logout-btn").onclick = function(){
+    sessionStorage.clear();
+    btnDiv.innerHTML = `<p color=red> Successully logged out</p>`
```
Comment (emmayuan25) [inline src/main/webapp/script.js]: same here

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -11,15 +11,15 @@ if(username === "" || username === null){
...
+    window.setTimeout(function() {
```
Comment (emmayuan25) [inline src/main/webapp/script.js:22]: nit: I think it's more common to do an alert("You've logged out") to inform users that they've logged out, and takes them back to index.html after they close the alert. An alert is more noticeable, but the message works as well.

**Review by emmayuan25 — COMMENTED** [review]
I like this feature!

File: src/main/webapp/script.js (hunk 3 above)
Comment (xiaoalem) [reply src/main/webapp/script.js:22]: Good idea! I've edited it!

**Review by emmayuan25 — APPROVED** [review]
LGTM

**Pull request merged**